DCode.Web/App_Data/Oct_31.sql

```diff
@@ -0,0 +1,399 @@
+CREATE DATABASE  IF NOT EXISTS `dcode` /*!40100 DEFAULT CHARACTER SET utf8 */;

```

Comment:
Please copy the current sql script and make modifications - it will be easy for maintenance instead of simply generating db backup file